x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_summary_view.tsx

```diff
@@ -81,11 +86,20 @@ const EnrichmentDescription: React.FC<ThreatSummaryItem['description']> = ({
           </RightMargin>
         </>
       )}
+      <ActionCell
+        data={(data ?? { field: fieldName }) as EventFieldsData}
```

Comment:
Can we simply build an appropriate `EventFieldsData` from the existing parameters and what we know about these fields? I suspect that we don't need to pass `data` through all these layers like this.

Reply:
I am seeking @angorayc's guidance on how to approach this piece as I haven't been able to find an implementation that utilizes `EventFieldsData` type (with no casting)

Reply:
It might require narrowing `ActionCell`'s props to just what it uses; it looks like only a few properties from `EventFieldsData` are used in there.